client/stats/sender.go

```diff
@@ -104,3 +135,26 @@ func (rss *RemoteStatsSender) send() error {
 		rss.signer,
 	)
 }
+
+// Subscribe subscribes the sender on the bus for relevant events
+func (rss *RemoteStatsSender) Subscribe(bus client.EventSubscriptionKeeper) {
+	bus.Subscribe(string(connection.StateEvent), rss.consumeStateEvent)
```

Comment:
`bus.Subscribe` returns an error, should we check it somehow?

Reply:
These will now be checked in DI.